docs/source/edge_networks.rst

```diff
@@ -0,0 +1,66 @@
+Support for Edge Networks
+=========================
+
+Recptor out-of-the-box has the ability to support complicated networking environments including edge networks.
+
+Consider the following environment:
+
+.. image:: edge.png
+
+Configurable-Items
+^^^^^^^^^^^^^^^^^^
+
+Receptor encapsulates the concepts of `below-the-mesh` and `above-the-mesh` connections. Please refer to :doc:`tls` for a better understanding of these networking layers.
+
+If a particular node in a network has higher than normal latency, we allow the users to define a finely-grained idle connection timeout value for any given Receptor node. This will help Receptor keep `below-the-mesh` tcp connections alive. Receptor will attempt to reconnect to a timed-out node, to a maximum retry limit and if this retry limit is surpassed, Receptor will drop the connection from the routing table.
+
+If a connection timeout occurs, the users can expect to see a message like this in their receptor logs.
+
+.. code::
+
+    DEBUG 2022/04/07 12:48:56 Sending initial connection message
+    ERROR 2022/04/07 12:48:56 Backend sending error read tcp 10.26.5    0.239:27199->10.102.21.131:35024: i/o **timeout**
+
+To circumvent this scenario from happening, users can leverage the `maxidleconnectiontimeout` parameter in their configuration files.
+
+`maxidleconnectiontimeout` A user-defined parameter in the configuration file that will set the `below-the-mesh` tcp connection timeout.
+
+The configuration files for the diagram above are listed below.
+
+foo.yml
+
+.. code-block:: yaml
+
+    ---
+    - node:
+        id: foo
+        maxidleconnectiontimeout: 60s
```

Comment:
all 3 of these nodes would need the same `maxidleconnectiontimeout`. Because the monitorConnectionAging running on bar will still timeout the connection to foo after 21 seconds (default). So the true timeout is the minimum of this value for a set of 2 peers